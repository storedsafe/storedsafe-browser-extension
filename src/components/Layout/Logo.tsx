import React from 'react';
import logo from './logo.png';
import './Logo.scss';

export const Logo: React.FunctionComponent = () => (
  <img src={logo} alt="StoredSafe" className="logo" />
);
