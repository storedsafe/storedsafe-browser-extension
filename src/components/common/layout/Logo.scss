@import '../../../styles/variables.scss';

.logo {
  display: block;
  max-height: 100%;
  box-sizing: border-box;
}
