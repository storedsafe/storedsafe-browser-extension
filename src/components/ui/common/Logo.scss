@import '../../../variables';

.logo {
  display: block;
  max-height: 100%;
}
