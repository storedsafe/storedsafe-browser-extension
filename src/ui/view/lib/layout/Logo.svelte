<script>
  import { getMessage, LocalizedMessage } from "../../../../global/i18n";
</script>

<style>
  img {
    max-width: 90%;
    max-height: 2.2em;
    margin: auto;
    user-select: none;
  }
</style>

<img src="/assets/logo.svg" alt={getMessage(LocalizedMessage.IMG_LOGO_ALT)} />
