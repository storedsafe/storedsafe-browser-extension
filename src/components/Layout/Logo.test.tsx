import React from 'react';
import { shallow } from 'enzyme';
import { Logo } from './Logo';

test('<Logo />', () => {
  const wrapper = shallow(<Logo />);
  expect(wrapper).toMatchSnapshot();
});
