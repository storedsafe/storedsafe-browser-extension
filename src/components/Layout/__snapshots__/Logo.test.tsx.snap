// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> 1`] = `ShallowWrapper {}`;
